module/config.mjs

```diff
@@ -538,6 +547,18 @@ preLocalize("itemRarity");
 
 /* -------------------------------------------- */
 
+/**
+ * The limited use periods that support a recovery formula.
+ * @enum {string}
+ */
+DND5E.limitedUseFormulaPeriods = {
+  charges: "DND5E.Charges",
+  dawn: "DND5E.Dawn",
+  dusk: "DND5E.Dusk"
+};
```

Comment:
Might be nice to display these in an `<optgroup>` in the drop-down on the sheet, but not a requirement. It can be follow-up work.